host/4/bullseye/amd64/python/python311/python311-slim.Dockerfile

```diff
@@ -0,0 +1,95 @@
+    unzip /$EXTENSION_BUNDLE_FILENAME_V4 -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V4 && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME_V4 &&\
+    find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
+
+FROM mcr.microsoft.com/mirror/docker/library/python:3.11-slim as python
+
+# Install Python dependencies
+RUN apt-get update && \
+    apt-get install -y wget apt-transport-https curl gnupg locales && \
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
+    # Needed for libss1.0.0 and in turn MS SQL
+    echo 'deb http://security.debian.org/debian-security stable-security main' >> /etc/apt/sources.list && \
+    # install necessary locales for MS SQL
+    echo 'en_US.UTF-8 UTF-8' > /etc/locale.gen && \
+    locale-gen && \
+    apt-get update && \
+    ACCEPT_EULA=Y apt-get install -y unixodbc msodbcsql17 mssql-tools &&\
+    apt-get install -y --no-install-recommends ca-certificates \
+    libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g &&\
+    apt-get install -y libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb binutils\
+    binutils libgomp1 libc-dev liblttng-ust0 && \
+    rm -rf /var/lib/apt/lists/*
+    # MS SQL related packages: unixodbc msodbcsql17 mssql-tools
+    # .NET Core dependencies: --no-install-recommends ca-certificates libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g
+    # OpenCV dependencies:libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb
+    # binutils: binutils
+    # OpenMP dependencies: libgomp1 && \
+    # Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157 : libc-dev
+    # Azure ML dependencies: liblttng-ust0
+
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
+ARG HOST_VERSION
+
+COPY --from=runtime-image ["/azure-functions-host", "/azure-functions-host"]
+COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
+COPY --from=runtime-image [ "/workers/python/3.11/LINUX", "/azure-functions-host/workers/python/3.11/LINUX" ]
+COPY --from=runtime-image [ "/workers/python/worker.config.json", "/azure-functions-host/workers/python" ]
+COPY --from=python [ "/usr", "/usr" ]
+COPY --from=python [ "/lib", "/lib" ]
+COPY --from=python [ "/lib64", "/lib64" ]
+
+ENV LANG=C.UTF-8 \
+    ACCEPT_EULA=Y \
+    AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=python \
+    ASPNETCORE_URLS=http://+:80 \
+    DOTNET_RUNNING_IN_CONTAINER=true \
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    HOST_VERSION=${HOST_VERSION} \
+    ASPNETCORE_CONTENTROOT=/azure-functions-host
+
+ENV FUNCTIONS_WORKER_RUNTIME_VERSION=3.11
+
+CMD [ "/azure-functions-host/Microsoft.Azure.WebJobs.Script.WebHost" ]
```

Comment:
And new line character at the end of the file. For all the files which don't have it.

Reply:
updated. thanks